mission7/queries.sql

```diff
@@ -0,0 +1,191 @@
+/*
+  1. 내 정보 업데이트 하기
+  - 닉네임을 "test"로 업데이트
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+UPDATE users
+SET nickname = 'test'
+WHERE id = 1;
+
+
+/*
+  2. 내가 생성한 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+SELECT 
+  id, name, price, created_at
+FROM products
+WHERE user_id = 1
+ORDER BY created_at DESC
```

Comment:
같은 created_at을 방지하고자 `id`를 타이브레이커로 활용할 수 있어요.
이 문제는 별거아닌 것 처럼 보이지만, 일관된 정렬을 보장하지 않아서, 페이징 중복/누락 등이 발생할 수 있습니다. 

다른 정렬 쿼리에도 동일하지만, 대표적으로 여기 남겨두겠습니다~!

```suggestion
ORDER BY created_at DESC, id DESC
```